include/crud-terms.php

```diff
@@ -307,27 +307,25 @@ private function get_slug_separator() {
 	 * @return string Slug with a language suffix if found.
 	 */
 	public function set_pre_term_slug( $slug, $taxonomy ) {
-		if ( ! empty( $slug ) ) {
-			return $slug;
+		if ( ! $slug ) {
```

Comment:
Since `$slug` is not a boolean, I would prefer to see `empty()`, but no obligations:
```suggestion
		if ( empty( $slug ) ) {
```